### PR TITLE
Revise <p> style decisions

### PR DESCRIPTION
**Pull request by EnnuiL**
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
i need to review this myself

### CODE REVIEW

**Comment by eristhea** [comment]
@EnnuiL I'd prefer if we split this into two, the \<p> fixes make it very difficult to sort out the dictionary changes

edit: huh I did not know github comments supported html

**Comment by EnnuiL** [comment]
> @EnnuiL I'd prefer if we split this into two, the \<p\> fixes make it very difficult to sort out the dictionary changes

heck

**Comment by eristhea** [comment]
I mean you can look at the commits but two prs would be cleaner :p

**Comment by EnnuiL** [comment]
This is still a draft due to aaaaaaaa

File: mappings/net/minecraft/obfuscate/DontObfuscate.mapping [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-	COMMENT
-	COMMENT <p>Its behavior appears as follows:
+	COMMENT <p>
+	COMMENT Its behavior appears as follows:
 	COMMENT <ul>
 	COMMENT <li>The annotation itself is not obfuscated.</li>
 	COMMENT <li>If a class is annotated, it is not obfuscated. It's not yet clear if its
 	COMMENT members will always become deobfuscated as well.</li>
 	COMMENT <li>If a member is annotated, it and its containing class is not obfuscated,
 	COMMENT but other members in the same class may stay obfuscated.</li>
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/obfuscate/DontObfuscate.mapping:11]:
```suggestion
	COMMENT 	<li>The annotation itself is not obfuscated.</li>
	COMMENT 	<li>If a class is annotated, it is not obfuscated. It's not yet clear if its
	COMMENT members will always become deobfuscated as well.</li>
	COMMENT 	<li>If a member is annotated, it and its containing class is not obfuscated,
	COMMENT but other members in the same class may stay obfuscated.</li>
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Comment by EnnuiL** [comment]
I'll close this PR due to the sheer unmaintainability of it; this might be need to be done in many PRs

**Comment by eristhea** [comment]
that's what aurora has been doing for ages :p

**Pull request closed without merging**